siliconcompiler/schema/schema_cfg.py

```diff
@@ -1260,17 +1260,25 @@ def schema_datasheet(cfg, name='default', mode='default'):
             example=[
                 "cli: -datasheet_package_type 'abcd bga'",
                 "api: chip.set('datasheet', 'package', 'abcd', 'type', 'bga')"],
-            schelp="""Package type specified on a named package basis.""")
+            schelp="""Package type.""")
 
     scparam(cfg, ['datasheet', 'package', name, 'footprint'],
-            sctype='str',
+            sctype='[file]',
             shorthelp="Datasheet: package footprint",
-            switch="-datasheet_package_footprint 'name <str>'",
+            switch="-datasheet_package_footprint 'name <file>'",
+            example=[
+                "cli: -datasheet_package_footprint 'abcd ./soic8.kicad_mod'",
+                "api: chip.set('datasheet', 'package', 'abcd', 'footprint', './soic8.kicad_mod')"],
+            schelp="""Package footprint file.""")
+
+    scparam(cfg, ['datasheet', 'package', name, '3dmodel'],
+            sctype='[file]',
+            shorthelp="Datasheet: package 3D model",
+            switch="-datasheet_package_3dmodel 'name <file>'",
             example=[
-                "cli: -datasheet_package_footprint 'abcd soic8'",
-                "api: chip.set('datasheet', 'package', 'abcd', 'footprint', 'soic8')"],
-            schelp="""Package footprint name. The name of the footprint can be a standard
-            footprint name or a reference designator from a footprint library.""")
+                "cli: -datasheet_package_3dmodel 'abcd ./soic8.step'",
+                "api: chip.set('datasheet', 'package', 'abcd', '3dmodel', './soic8.step')"],
+            schelp="""Package 3D model file. Common 3D model standards include STEP and WRL.""")
```

Comment:
step and wrl? if they are acronyms, they should be spelled out and we should make sure they appear in our glossery